apps/src/applab/applab.js

```diff
@@ -1208,7 +1208,7 @@ Applab.renderVisualizationOverlay = function() {
   // Change cursor rule for divApplab - appropriate here because sometimes
   // the overlay is sort of a 'virtual' cursor for us.
   [divApplab, designModeViz].forEach(function (div) {
-    div.style.cursor = Applab.isRunning() ? '' : 'crosshair';
+    $(div).toggleClass('withCrosshair', !Applab.isRunning());
```

Comment:
designModeViz is only shown when we are not running, so it's probably not necessary to apply `withCrosshair` here.